assignments/basic-part4-ass2/assignment.js

```diff
@@ -0,0 +1,41 @@
+const randomNumber = Math.random(); // produces random number between 0 (including) and 1 (excluding)
```

Comment:
folder naming should be
../basic-part-4-assessment-2/..

NEVER like part4

"ass2", are you sure? translate(ass) → дупа
be careful with short naming, try to avoid